README.md

```diff
@@ -9,12 +9,23 @@ Part of the Foreman installer: <https://github.com/theforeman/foreman-installer>
 
 | Module version | Proxy versions | Notes                                           |
 |----------------|----------------|-------------------------------------------------|
-| 5.x            | 1.16 and newer |                                                 |
+| 5.x            | 1.16 and newer | See compatibility notes here for 1.16-1.18      |
```

Comment:
@ekohl Version 5? What about 6,7,8 and 9?

Reply:
This should be 5.x and newer. The breaking changes have been for different reasons than compatibility. Untested, but I think 9 should still allow managing 1.16. It'd be good to be explicit about this though.